common-lib/src/main/scala/com/gu/mediaservice/lib/cleanup/SupplierProcessors.scala

```diff
@@ -209,7 +219,7 @@ object GettyCreditParser extends ImageProcessor with GettyProcessor {
   def knownGettyCredits(image: Image, credit: String): Image =
     gettyCredits.find(_.toLowerCase == credit.toLowerCase) match {
       case collection @ Some(_) => image.copy(
-        usageRights = gettyAgencyWithCollection(collection)
+        usageRights = gettyAgencyWithCollection(collection).getOrElse(NoRights)
```

Comment:
I'm a little out of date on Grid stuff, so this might be a pointless comment...

Is it correct to return `NoRights` here as the function name suggests it should return a `Credit`?